src/config/texts.js

```diff
@@ -275,6 +275,11 @@ export const texts = {
       expired: 'Abgelaufen'
     }
   },
+  contentList: {
+    title: 'Mach Meldung in drei Schritten',
+    description:
+      'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.'
+  },
```

Comment:
please do not implement backup texts. if there should be content, it needs to be configured. we can save code and logics with not having fallbacks here.

Reply:
Yes, you are right. 👍

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/958/commits/c5071d1275e9c9450bb63ab91c3c2de325370162